src/data/fintype/sigma.lean

```diff
@@ -0,0 +1,32 @@
+/-
+Copyright (c) 2017 Mario Carneiro. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro
+-/
+import data.fintype.basic
+import data.finset.sigma
+
+
+/-!
+# fintype instances for sigma types
+-/
```

Comment:
There's an argument that those structure files should be merged with the corresponding ones under `finset`. The ones under `finset` are basically only imported by the `fintype` ones, and they are a continuation of each other.